janusgraph-cassandra/src/test/java/org/janusgraph/CassandraStorageSetup.java

```diff
@@ -137,7 +137,7 @@ public static String cleanKeyspaceName(String raw) {
         Preconditions.checkNotNull(raw);
         Preconditions.checkArgument(0 < raw.length());
 
-        if (48 < raw.length() || raw.matches("[^a-zA-Z_]")) {
+        if (48 < raw.length() || raw.matches("^.*[^a-zA-Z0-9_].*$")) {
```

Comment:
Do we actually need the additional complexity of `^.*[^a-zA-Z0-9_].*$` or can we just use `[^a-zA-Z0-9_]` here? Seems like the test is to find whether or not there exists an invalid character, so we don't need to match the full string.